finesse/hardware/stepper_motor/st10_controller.py

```diff
@@ -275,6 +275,9 @@ def _home_and_reset(self) -> None:
         # Tell the controller that this is step 0 ("set variable SP to 0")
         self._write_check("SP0")
 
+        # Make sure the motor has stopped
+        self.wait_until_stopped(20.0)
```

Comment:
Is this 20 seconds? This seems like quite a long time, but you'd know better than me! I'm just wondering if it's possible to calculate the maximum time this could take based on how fast it rotates and the maximum possible angle (270 degrees?)

Reply:
Yup! It is a conservative value, but it's not likely that we will ever actually wait that long, unless something v specific goes wrong. I'm not sure how fast it actually moves off the top of my head although I suspect 10s might be long enough.

Do you have a better idea @jonmurray?